### PR TITLE
Feed tool docs (SYN-1650)

### PR DESCRIPTION
**Pull request by rakuy0**

### CODE REVIEW

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 1]
```diff
@@ -11,7 +11,235 @@
     "Synapse Tools - feed\n",
     "====================\n",
     "\n",
-    "TBD"
+    "The Synapse ``feed`` tool is a way to ingest data exported from one Cortex into another Cortex. Users should be familiar with both the Synapse data model (:ref:`data-model-terms` et al.) as well as the Synapse concepts such as packed nodes and splices in order to use and understand the ``feed`` tool effectively.\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:14]: "....as well as the Synapse concepts such as...." -> "...as well as Synapse concepts..."

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 2]
```diff
@@ -11,7 +11,235 @@
...
+    "\n",
+    "\n",
+    "Syntax\n",
+    "------\n",
+    "The ``feed`` tool is exected from an operating system command shell. The command usage is as follows (line is wrapped for readability):\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:19]: "...exected..." -> "....executed...."

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 3]
```diff
@@ -11,7 +11,235 @@
...
+    "\n",
+    "::\n",
+    "\n",
+    "  usage: synapse.tools.feed [-h] (--cortex CORTEX | --test) [--debug] [--format FORMAT] [--modules MODULES]   \n",
+    "    [--chunksize CHUNKSIZE] [--offset OFFSET] [files ...]\n",
+    "\n",
+    "Where:\n",
+    "- ``-h`` displays detailed helped and these command line options\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:27]: "displays detailed helped...." -> "displays detailed help..."

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 4]
```diff
@@ -11,7 +11,235 @@
...
+    "- ``CORTEX``  specifies the path to the Cortex where the data should be ingested.\n",
+    "\n",
+    "  - See :ref:`cortex-connect` for the format to specify a path to a Cortex.\n",
+    "  - Cortex paths can also be specified using aliases defined in the user's ``.syn/aliases.yaml`` file.\n",
+    "\n",
+    "- ``--test`` means to perform ingest against a temporary, local Cortex instead of a live cortex, for test or validation\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:33]: "...to perform ingest...." -> "....to perform the ingest..."
"...for test or validation" -> "...for testing or validation"

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 5]
```diff
@@ -11,7 +11,235 @@
...
+    "  \n",
+    "  - When using a temporary Cortex, you do not need to provide a path.\n",
+    "  \n",
+    "- ``--debug`` specifies to drop into an interactive prompt to inspect the state of the Cortex post-ingest. \n",
+    "\n",
+    "  - See :ref:`_syn-ref-cmdr` for how to use the interactive prompt\n",
+    "  \n",
+    "- ``FORMAT`` specifics the format the input files are in. \n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:41]: "specifics the format..." -> "specifies the format..."
"...the format the input files are in" -> "...the format of the input files."

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 6]
```diff
@@ -11,7 +11,235 @@
...
+    "\n",
+    "  - Currently, only the values \"syn.nodes\", \"syn.splices\", and \"syn.nodeedits\" are supported.\n",
+    "  - Defaults to \"syn.nodes\" if not specified\n",
+    "  \n",
+    "- ``MODULES`` specifies a path to a Synapse CoreModule class that will be loaded into the temporary Cortex.\n",
+    "\n",
+    "  - This option has no effect if the ``--test`` option is not specified\n",
+    "  - For more on Core Modules, see :ref:`_dev_cortex_quickstart`\n",
+    "- ``CHUNKSIZE`` specifies how many lines or chunks of data to read at a time from the gives files.\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:50]: "...from the gives file" -> "....from the given files."

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 7]
```diff
@@ -11,7 +11,235 @@
...
+    "\n",
+    "  - Defaults to 1000 if not specified\n",
+    "\n",
+    "- ``OFFSET`` specifies how many chunks of data to skip over from (starting at the beginning)\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:54]: "...to skip over from..." -> "...to skip over...."

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 8]
```diff
@@ -11,7 +11,235 @@
...
+    "  \n",
+    "Ingest Examples - Overview\n",
+    "--------------------------\n",
+    "\n",
+    "The ``feed`` tool \n",
+    "\n",
+    "Ingest Example 1\n",
+    "++++++++++++++++\n",
+    "\n",
+    "This examples demonstrates loading a set of nodes via the ``feed`` tool with the \"syn.nodes\" format option. The nodes are of a variety of types, and are encoded in a json lines (jsonl) format.\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:69]: "This examples....." -> "This example..."

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 9]
```diff
@@ -11,7 +11,235 @@
...
+    "  [[\"file:bytes\", \"sha256:ffd19426d3f020996c482255b92a547a2f63afcfc11b45a98fb3fb5be69dd75c\"], {\"iden\": \"137fd16d2caab221e7580be63c149f83a11dd11f10f078d9f582fedef9b57ad5\", \"tags\": {\"my\": [null, null], \"my.cool\": [null, null], \"my.cool.tag\": [null, null]}, \"props\": {\".created\": 1625070470041, \"sha256\": \"ffd19426d3f020996c482255b92a547a2f63afcfc11b45a98fb3fb5be69dd75c\", \"md5\": \"be1bb5ab2057d69fb6d0a9d0684168fe\", \"sha1\": \"57d13f1fa2322058dc80e5d6d768546b47238fcd\", \"size\": 16}, \"tagprops\": {}, \"nodedata\": {}, \"path\": {}}]\n",
+    "\n",
+    "\n",
+    "**Verifying the Data:**\n",
+    "\n",
+    "Typically, users will want to double check the data they have before loading it into a production Cortex. The ``feed`` tool allows us to perform an ingest our of nodes file against an empty, emphemeral Cortex, so that we can check what nodes get created before slamming them into production. To load ``testnodes.jsonl`` into an emphemeral Cortex and drop into a prompt to explore the ingested nodes, run:\n",
```
Comment (therealsilence) [inline docs/synapse/userguides/syn_tools_feed.ipynb:85]: "...emphemeral..." -> "...ephemeral..." (2x)

+1 on "before slamming them into production"

**Review by therealsilence — COMMENTED** [review]
Spelling / grammar corrections. :-)

File: docs/synapse/userguides/syn_tools_feed.ipynb [hunk 10]
```diff
@@ -11,7 +11,235 @@
...
+    "  - For more on Core Modules, see :ref:`_dev_cortex_quickstart`\n",
+    "- ``CHUNKSIZE`` specifies how many lines or chunks of data to read at a time from the given files.\n",
+    "\n",
+    "  - Defaults to 1000 if not specified\n",
+    "\n",
+    "- ``OFFSET`` specifies how many chunks of data to skip over (starting at the beginning)\n",
+    "\n",
+    "- ``files`` is a series of file paths containing data to load into the Cortex (or temporary Cortex)\n",
+    "\n",
+    "  - Every file must be either json-serialized data, msgpack-serialized data, yaml-serialzed data, or a \n",
```
Comment (Cisphyx) [inline docs/synapse/userguides/syn_tools_feed.ipynb:58]: missing an 'i' in yaml-serialzed

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**